src/command-validation-support/src/main/java/engineering/everest/lhotse/axon/command/validators/OrganizationStatusValidator.java

```diff
@@ -24,8 +28,10 @@ public void validate(OrganizationStatusValidatableCommand command) {
         try {
             organization = organizationsReadService.getById(command.getOrganizationId());
         } catch (NoSuchElementException e) {
-            throw new IllegalStateException(String.format("Organization %s does not exist", command.getOrganizationId()), e);
+            throw new TranslatableIllegalStateException(ORGANIZATION_DOES_NOT_EXIST, e, command.getOrganizationId());
```

Comment:
Should this use the new `TranslatableExceptionFactory.throwForKey` method as well?

Reply:
Ahh, missed one. Thanks :heart: